tests/ui/unit/react/groups.tests.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { mount } from 'enzyme';
 import JobGroup from '../../../../ui/job-view/JobGroup';
+import { thEvents } from "../../../../ui/js/constants";
```

Comment:
Whilst you're here, could you add an extra space between the relative imports and those for third party packages? :-)